libs/ui/src/components/NeoButton/NeoButton.vue

```diff
@@ -81,17 +83,86 @@ defineProps<{
     &:hover {
       background: $white !important;
     }
+    &:hover {
+      background: white;
+    }
+    &.o-btn--disabled {
+      opacity: unset;
+      background-color: $shade;
+      color: $k-grey;
+      border-color: $k-grey;
+    }
   }
 
   &--k-blue {
     background-color: $k-blue;
     &:hover {
       background: $white !important;
     }
+    &:hover {
+      background: white;
+    }
+    &.o-btn--disabled {
+      opacity: unset;
+      background-color: $shade;
+      color: $k-grey;
+      border-color: $k-grey;
+    }
   }
 
   &.no-shadow {
-    box-shadow: none;
+    box-shadow: none !important;
+  }
+}
+
+.dark-mode {
+  .is-neo {
```

Comment:
same stuff has been done in #4575, there may be a conflict here

Reply:
it's a result of a merge conflict i resolved.
I'll have a closer look

Reply:
css is ok now. quite sure